apps/system/test/unit/edge_swipe_detector_test.js

```diff
@@ -577,7 +565,7 @@ suite('system/EdgeSwipeDetector >', function() {
         var recvEvents = verticalSwipe();
 
         var call = fwSpy.secondCall;
-        assert.equal(call.args[0], recvEvents[7]);
+        assert.equal(call.args[0], recvEvents[8]);
```

Comment:
I'm pretty sure we shouldn't change this. I agree that this is hard to read,
But I'm pretty sure this test will never pass since it asserts that the second and third call to forward got the same argument (`recvEvents[8]` instead of `recvEvents[7]` then `recvEvents[8]`)